docs/usage/ipv6.md

```diff
@@ -1,38 +1,22 @@
 # DualStack Support for Gardener GCP Extension
 
-This document provides an overview of DualStack support for the Gardener GCP extension, detailing its functionality, requirements, and implementation specifics. The document also clarifies the differences between provisioning methods and the unique components required for DualStack clusters.
+This document provides an overview of DualStack support for the Gardener GCP extension.
+Furthermore it clarifies which components are utilized, how the infrastructure is setup and how a DualStack cluster can be provisioned.
 
 ---
```

Comment:
Might be personal preference, but I can live without the multiple separators in this document. Therefore, I propose to drop all of them, e.g.

```suggestion
```